app/helpers/meta_tags_helper.rb

```diff
@@ -0,0 +1,10 @@
+module MetaTagsHelper
```

Comment:
there's no need to create more and more helper modules.. it's bad for performance,
meta tags are app components, so `ApplicationHelper` is not really bad place for this helper